tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cpu.yml

```diff
@@ -76,6 +76,8 @@ stages:
               --volume $HOME/.onnx:/home/onnxruntimedev/.onnx \
               -e NIGHTLY_BUILD \
               -e BUILD_BUILDNUMBER \
+              -e ORT_DISABLE_PYTHON_PACKAGE_LOCAL_VERSION \
+              -e DEFAULT_TRAINING_PACKAGE_DEVICE \
```

Comment:
Do we need both these variables? Can we keep just one? This PR is OK, we can do the refactoring later. I am just curious whether we can keep just 1.